index.html

```diff
@@ -8,7 +8,7 @@
 	<div class="container">
 		<div class="box">
 			<input type="password" class="input-field" placeholder="Enter new password">
```

Comment:
An id equal to password needed to be added as the function is trying to get an element with Id password.